### PR TITLE
chore: modify build:csmith to wireit

### PR DESCRIPTION
**Pull request by yimingjfe**
# PR Details

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] I have updated changeset
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/generator/plugins/generator-plugin/package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "prepublishOnly": "only-allow-pnpm && pnpm build --platform",
     "new": "modern new",
     "build": "wireit",
-    "build:csmith": "csmith-tools build",
+    "build:csmith": "wireit",
```
Comment (chenjiahan) [inline packages/generator/plugins/generator-plugin/package.json:31]: 这个文件需要添加 wireit 配置不

Comment (chenjiahan) [inline packages/generator/plugins/generator-plugin/package.json:31]: 这个文件需要添加 wireit 配置不

File: packages/toolkit/create/package.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
     "prepublishOnly": "only-allow-pnpm",
     "new": "modern new",
     "build": "wireit",
-    "build:csmith": "csmith-tools build",
+    "build:csmith": "wireit",
```
Comment (chenjiahan) [inline packages/toolkit/create/package.json:42]: 同上

Comment (chenjiahan) [inline packages/toolkit/create/package.json:42]: 同上

File: packages/generator/plugins/generator-plugin/package.json (hunk 1 above)
Comment (yimingjfe) [reply packages/generator/plugins/generator-plugin/package.json:31]: done.

File: packages/toolkit/create/package.json (hunk 2 above)
Comment (yimingjfe) [reply packages/toolkit/create/package.json:42]: done.

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**